app/controllers/webhooks_controller.rb

```diff
@@ -36,4 +36,27 @@ def gocardless
 
     head(:ok)
   end
+
+  def adyen
+    signature = adyen_params.dig("additionalData", "hmacSignature")
+
+    result = PaymentProviders::AdyenService.new.handle_incoming_webhook(
+      organization_id: params[:organization_id],
+      body: adyen_params
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)